Makefile

```diff
@@ -347,6 +348,19 @@ e2e: # Run E2E integration tests.
 	./hack/verify.sh integration
 .PHONY: e2e
 
+deploy-prometheus: ## Deploy Prometheus
```

Comment:
please move kube-prometheus set up to https://github.com/sustainable-computing-io/local-dev-cluster as where we set up prometheus operator. hence, I am confusing if we set up two prometheus on the kind cluster?

Reply:
OK will do

Reply:
I didn't think/see cluster-up deploying prometheus unless I explicitly set env vars (which I probably need to remember to unset later)...

With this change I was just trying to make sure I can spin up a quick kind cluster for validation and remove the test env without having to make any changes on my host.

I will try to combine the two concepts and move/update the prometheus script in the cluster local side

Reply:
We can add something like this in `hack/cluster.sh` to make sure Prometheus and grafana both deployed as [local-dev-cluster](https://github.com/sustainable-computing-io/local-dev-cluster) supports deployment of both
```
declare -r GRAFANA_ENABLE=${GRAFANA_ENABLE:-false}
....
....
export GRAFANA_ENABLE
```
In `cluster-up` target:
```
...
GRAFANA_ENABLE=$(GRAFANA_ENABLE) \
KIND_WORKER_NODES=$(KIND_WORKER_NODES) \
./hack/cluster.sh up
```